src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -433,7 +433,8 @@ export default compose(
       overallSurvivalData,
       survivalDataLoading,
     }) => (
-      !isEqual(currentAnalysis, nextCurrentAnalysis) ||
```

Comment:
this change causes overall survival to update ONLY if you switch to a new analysis or case set.

previously, it would update if you made custom bins in the gender card, even though that has nothing to do with overall survival.